src/applicationInsightsClient.ts

```diff
@@ -25,12 +29,13 @@ export class ApplicationInsightsClient {
                 "Connection String not found, please provide it before starting Application Insights SDK."
             );
         }
-        if (!this._config.getDisableStatsbeat()) {
-            this._statsbeat = new Statsbeat(this._config);
-            this._statsbeat.enable(true);
-        }
+        this._getStatsbeatInstrumentations();
+        this._getStatsbeatFeatures();
+
+        // Statsbeat enable/disable is handled from within the Statsbeat class
+        this._statsbeat = new Statsbeat(this._config);
         this._metricHandler = new MetricHandler(this._config);
-        this._traceHandler = new TraceHandler(this._config, this._metricHandler);
+        this._traceHandler = new TraceHandler(this._config, this._metricHandler, this._statsbeatInstrumentations, this._statsbeatFeatures);
```

Comment:
@hectorhdzg Would passing the Instrumentations and Features to the Metrics exporters as well make sense? Seems like this would create a situation where both the traceExporter and the metricsExporter create multiple feature/instrumentation statsbeats when this kind of statsbeat is intended to be created once per instance of the application. All depends on if it's possible for a standalone metric to be created, as in that case we'd want to make sure the feature/instrumentation statsbeat is still generated.